### ISSUE TITLE
VPC and Efflux transporter question

### ISSUE THREAD

**Issue by ms-12345** [Open-Systems-Pharmacology/Forum#207]
Hello,

I have a few questions-

1.) As I understand it currently, the VPC, Pred vs. DV, Res vs. Time and residual distribution plots is available in PK-SIM only after certain parameters are identified based on observed data, ie, after generating the optimal value.

What if I would like to generate these plots based on the current simulation (input data that I have provided)  without the minimization procedure being carried out? Is it possible to get the residual plots, DV vs. PRED, VPC etc for my current model?

2.) This question is related to efflux transporters.

Supposing the publication for the P-gp efflux gives me Jmax (pmol/min) and Km (uM) values for Caco-2 cell monolayers, how would I scale it for use in PK-SIM?

3.) To model the effect of additional routes of elimination of a drug, (extracorporeal clearance) will I need to define a new compartment in Mobi? Is there an example I can refer to for the creation of a new eliminating compartment?

4.) Given the change in albumin concentrations, how should the plasma protein scale factor be modified to account for increased / decreased Fu? 

Thank you for your help!

**Comment by PavelBal** [comment]
1) In the "Configuration"-tab of the Parameter Identification, change the value of "Maximum Number of Iterations" to 0 and run. By doing this, no optimization is performed, but every simulation is run once and the results (like residual error, VPC etc) are generated.

**Comment by HannahBri** [comment]
@PavelBal  Thank you very much for your helpful comment. 
How can I export the data of the generated Pred vs. Obs plots?

**Comment by ms-12345** [comment]
Hi @PavelBal thank you for your help! 
Hi @HannahBri Just right click on the selected plot and click "Export to Excel", the data will open in excel.

**Comment by Yuri05** [comment]
>3.) To model the effect of additional routes of elimination of a drug, (extracorporeal clearance) will I need to define a new compartment in Mobi? Is there an example I can refer to for the creation of a new eliminating compartment?

You can incorporate a new elimination route in MoBi in several ways. The easiest way (especially if you are not interested in the amount of eliminated drug) would be to introduce a new _elimination reaction_ without any products.

S. example attached.
[UserDefinedElimination_Example.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/2290042/UserDefinedElimination_Example.zip)

Here a PBPK model (exported from PK-Sim) was extended with additional elimination reaction.
The reaction has no products (cleared drug is just ignored) and occurs (just as an example) only in _Venous Blood/Plasma_ (defined via **Container Criteria** of the reaction)

<img width="541" alt="aufnahme44" src="https://user-images.githubusercontent.com/25061876/44145589-b3886bd4-a07b-11e8-8a0e-b5158b123cfe.png">
<img width="400" alt="aufnahme45" src="https://user-images.githubusercontent.com/25061876/44148406-cd3717b8-a087-11e8-8d37-d22fb7693fe7.png">

**Comment by ms-12345** [comment]
Thank you @Yuri05 for the explanation!! I appreciate your help.

**Comment by Sheena311** [comment]
Hello @Yuri05 , 
How to integrate the passive diffusion clearance for a drug in PK-Sim?
Thank you in advance!

**Comment by msevestre** [comment]
@Sheena311 I'd suggest to ask your question in a new issue so that we do not get the content mixed up. As you can see, your question fell under the radar because of that
